### PR TITLE
Board add candidate component

### LINKED ISSUE #92: Extend the BoardPage and Add Candidates
## User story

**Who:** **As a** User Owner of a Board
**What:** **I want to**  be able to add new candidates
**Why:** **so that I can**  make sure that people consider all of the options and rank them accordingly to their preference

## Acceptance criteria

The page should look like this if the user is the Owner of the Board
Screenshot 2021-07-30 at 13 53 02

#### -> React Containers and Components 

- [ ] Extend the Board Page - https://github.com/HackYourFuture-CPH/simply-name.it/issues/74 and add funcionality to Candidate Input - https://github.com/HackYourFuture-CPH/simply-name.it/issues/9         
- [ ] and Add Button -  https://github.com/HackYourFuture-CPH/simply-name.it/issues/2 (probably needs some adaptation)

#### -> Endpoints

- [x] Endpoint to add a new candidate - https://github.com/HackYourFuture-CPH/simply-name.it/issues/91

### PR DESCRIPTION
**Pull request by Hani-far**
# Description

Two BoardAddCandidate components were created for PreDeadline and PostDeadline

Fixes #92 

# How to test?

npm run dev

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by Hani-far** [comment]
@senner007 Hi Niels,
I created two separate Board add candidate components for pre-deadline and post-deadline

File: src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidatePostDeadline.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './BoardAddCandidate.style.css';
+import Input from '../../../components/InputComponent/InputComponent';
+import AddButton from '../../../components/AddButton/AddButton.component';
+
+export default function BoardAddCandidate() {
```
Comment (senner007) [inline src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidatePostDeadline.js:6]: I think the Compponents should be named the same as the files they are in

**Review by senner007 — CHANGES_REQUESTED** [review]

File: src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidatePostDeadline.js (hunk 1 above)
Comment (Hani-far) [reply src/client/containers/BoardPageComponents/BoardAddCandidate/BoardAddCandidatePostDeadline.js]: yes, It's done now.

**Review by senner007 — APPROVED** [review]
✔

**Comment by Hani-far** [comment]
> ✔

Can I merge it now?

**Pull request merged**